STMReco/fcl/prolog.fcl

```diff
@@ -0,0 +1,31 @@
+#
+# prolog.fcl for fcl paramaters that will be used in STM modules
+#
+# Some notes:
+# - The sampling frequency in different for the HPGe and LaBr in the test beam. In the real experiment this will not be the case.
+#
+BEGIN_PROLOG
+
+STM : {
+
+  HPGe : {
+    samplingFrequency : 320.0520833313 # MHz - will probably end up in a DB somewhere
```

Comment:
Andy checked in with me before making this commit.  I feel it is part of my job to push back against making too many tables or making them too early because I don't want to take maintenance and efficiency lightly, since it is hard to measure and will sneak up on us.  We can talk more.  Once the decision is made, I'm ready to support.